### ISSUE TITLE
Add support for triple double-quote multiline strings in Python

### ISSUE THREAD

**Issue by bjlange** [stkb/vscode-rewrap#15]
Right now `rewrap` only supports multiline strings like this:
```
'''Here is a long
ass
string'''
```
but these are also valid in Python:
```
"""Yet another
long ass
string"""
```

**Comment by stkb** [comment]
Hi, I've added a fix to this. Just want to investigate #11 a little before pushing a new version; shouldn't take too long.

**Comment by bjlange** [comment]
Awesome, thanks for your speedy fix! I tried poking around on a fork of my own but you beat me to it; looks like I forgot the parens in the regex. Looking forward to the new version.

**Comment by stkb** [comment]
Yeah the string gets added into a much larger regex, so the parens are needed. Have just published v0.6.3 which contains the fix.
